tests/Integration/Behaviour/Features/Context/CommonFeatureContext.php

```diff
@@ -106,6 +107,16 @@ public function clearEntityManager()
         $this::getContainer()->get('doctrine.orm.entity_manager')->clear();
     }
 
+    /**
+     * @BeforeStep
+     *
+     * Clear Product prices cache at each step in order to get fresh data
+     */
+    public function clearProductPrices()
+    {
+        Product::flushPriceCache();
```

Comment:
Maybe it's a bit overkill to perform this clear at each step? (and in Common context which is used by many tests contexts) This could lead to unexpected issues, like the cache is cleared at each step in behat tests, but it's not the case in the real BO or FO which leads to different behaviour that we can't detect in the tests

Reply:
I agree but not doing it lead to unexpected issues too 😅. I think that clearing it between each step is closer to what really happens between each http request than not clearing.